app/src/main/java/com/example/vybe/VibeEvent.java

```diff
@@ -0,0 +1,53 @@
+package com.example.vybe;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.util.Date;
+
+public class VibeEvent implements Serializable {
+    private Vibe vibe;
+    private LocalDateTime dateTime;
```

Comment:
Can we get some documentation on LocalDateTime so me and Moe can update the Wiki, and understand what you guys decided?